### ISSUE TITLE
Skip to Content link becomes unavailable after use

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13555]
## Describe the defect
On each page, there is a Skip to Content link. However, if the user activates this (either mouse or keyboard), it becomes `hidden` and the user no longer has access to this.

This issue was reported in a recent audit from the 508 Office. Note from the report itself:

> Hiding the skip to content link when it doesn’t have focus is fine but hiding it from the screen reader completely after it is used is a defect.  I don’t know why anyone would do that.

## To Reproduce
1. Go to any page on VA.gov, for example: https://www.va.gov/health-care/about-va-health-benefits/
2. Start tabbing through the page immediately
3. You will see the Skip to content link appear once you focus on it
4. Hit enter to activate the link and jump down to the main content portion of the page
5. Now tab backwards through the page (SHIFT + Tab) to try and get back to the Skip to Content link.
6. Notice that this link never gets focused on, it never appears

## AC / Expected behavior

- [x] The Skip to content link should always be available, even after it has been activated.
- [x] Accessibility lead review

## Screenshots
The Skip to Content link doesn't appear because after being activated the `hidden` attribute is added to the element:
![image](https://user-images.githubusercontent.com/106678594/235522944-5d64940f-cf1c-4add-b43b-96b8442ba4da.png)

# Additional Info
It should ideally function like this: https://www.gov.uk/ - visually hidden if it is not focused on but available all the time to be focused on.

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by chriskim2311** [comment]
@laflannery This has made it to production. I have verified. Can you check on production and then we can close out this ticket thank you!

**Comment by laflannery** [comment]
Looks good to me @chriskim2311!
